src/main/resources/templates/home.html

```diff
@@ -14,11 +14,11 @@
                 <dt>Provided by:</dt>
                 <dd><th:block th:include="main.html :: organisation(${custodian}, ${branding})"></th:block></dd>
                 <dt>Total records:</dt>
-                <dd><a href="/records" th:text="${totalRecords}">Total records</a></dd>
+                <dd><a href="/records" th:utext="${totalRecords + '&lt;span class=&quot;visuallyhidden&quot;&gt; records&lt;/span&gt;'}">Total records</a></dd>
```

Comment:
What do you think of doing it this way?

```html
<a href="/records"><th:block th:text="${totalRecords}">Total</th:block> <span class="visuallyhidden"> records</span></a>
```
this avoids the `th:utext` and the html-in-html thing.